code/modules/antagonists/wizard/equipment/artefact.dm

```diff
@@ -220,6 +220,7 @@
 	w_class = WEIGHT_CLASS_TINY
 	var/list/spooky_scaries = list()
 	var/unlimited = 0
+	var/spec = /datum/species/skeleton
```

Comment:
Use descriptive variable names and autodoc your comments  
https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md

Reply:
thats what i said copycat

Reply:
I hereby maintainer decree that I said it first and any evidence in the contrary is a github bug